training_config/dialog_qa.json

```diff
@@ -108,11 +108,9 @@
         },
         "num_epochs": 30,
         "optimizer": {
-            "type": "adam",
```

Comment:
Did you mean to make this change?

Reply:
yes, for the camera ready we used sgd version as it was more stable.